modules/appeals_api/app/workers/appeals_api/appeal_received_job.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module AppealsApi
+  class AppealReceivedJob
+    include Sidekiq::Worker
+
+    def perform(opts)
```

Comment:
Let's [document the shape of the data we expect](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/doc_helpers.rb#L26-L30) from `opts`. 

Also, this has taken away two failure checks that we've been burned by before (incorrect/missing claimant data & blank template_id). Just because we're generally in control of both ends of the call to the job doesn't mean we should eschew sanity checking - I'd rather have the job "fail successfully" once with a log about a blank template Id than continually get repeated error alerts from VANotify whenever we send them a request w/o a template id.